release-deployment.md

```diff
@@ -138,20 +138,43 @@ There's nothing special about that. Each developer follows the above [Develop a
 1. Finish off the tasks in the release checklist. Once everything is done, close
    the release PR.
 
-**TBD: Discuss**
-Mike N: Long-lived release branches, yes/no?
-
-1. Create the release on Github from `master`.
+1. Delete the release branch on Github.
 
 ### Change in plan, pull a feature from a release
 
 **TBD: Discuss**
 Mike N: That probably means recreating the release branch, unless we have short-lived release branches
 
-### Change request
+### Supporting old releases
+
+In a release based project old releases can often need some maintenance. Critical bug fixes and strategic feature requests need to be supported on old releases. Old releases often become incompatible with the most recent versions of projects.
+
+In order to support old release versions gitflow has introduced the concept of support branches. Support branches are long living branches created to support major or minor versions of the project. Support branches do not get merged back into `master` or `develop` (this would cause major merge issues which are time consuming and error prone if attempted). Instead commits can be cherry picked from the support branch back into `develop`. 
+
+Support branches can be thought of as the `master` branch for old releases. Support branches for major releases should be named as `support-v<major>.x`. Support branches for minor releases should be named as `support-v<major>.<minor>.x`.
```

Comment:
So we intentionally leave the `x` as an `x`? In other words, the `x` is left as is in this naming convention, it is not meant to be replaced with a number?

Reply:
Yes we leave the `x` as it is a long living branch and the `x` changes for each release. So in astro we currently have a `support-v0.18.x` branch. The most recent release tagged off that branch is `0.18.6`.  Subsequent patches to `0.18` will be branched off of the branch, merged back into it and tagged for release. Does that make sense?

Reply:
Yup, makes sense!